cri/v1alpha2/cri.go

```diff
@@ -257,11 +258,15 @@ func (c *CriManager) RunPodSandbox(ctx context.Context, r *runtime.RunPodSandbox
 	}
 	id := createResp.ID
 
+	// init container log
+	containerLogMap := make(map[string]string)
+
 	// once sandbox container created, we are obligated to store it.
 	sandboxMeta := &SandboxMeta{
-		ID:      id,
-		Config:  config,
-		Runtime: config.Annotations[anno.KubernetesRuntime],
+		ID:              id,
+		Config:          config,
+		Runtime:         config.Annotations[anno.KubernetesRuntime],
+		ContainerLogMap: containerLogMap,
```

Comment:
could we use `ContainerLogMap: make(map[string]string)` to do initialization? since the `containerLogMap` is only used for assignment.